pkg/resources/k8sgpt.go

```diff
@@ -23,7 +23,6 @@ import (
 	"github.com/k8sgpt-ai/k8sgpt-operator/pkg/utils"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
-	v1 "k8s.io/api/core/v1"
```

Comment:
might seem pedantic, but there is no need to remove this alias, and revert to `corev1`

Reply:
Hey, I just want to understand why this was imported twice here. I saw this warning on my IDE and decided to work on it. Any specific reason why is there already.

Reply:
Actually, you are right, we import it twice.. ignore me, it's fine

Most likely, two different contributors added the same module, thanks for removing the duplication